pages/linux/chkconfig.md

```diff
@@ -0,0 +1,27 @@
+# chkconfig
+
+> Manage service runlevel.
```

Comment:
I assume that this command is part of a service manager of some description? I've got ubuntu with systemd - and it's not installed.

Perhaps we could expand this description to include the service manager that this works with? For example `Manage insert_service_manager_name_here service runlevels.` would work?

Reply:
@sbrl this is the service manager for Centos 6. I'm not sure about Red Hat/Fedora though as I'm not using any of them.

For latest Centos with systemd we can do something like `systemctl {enable|disable} <service_name>`. The equivalent on Centos 6 is `chkconfig <service_name> on`

Reply:
Ah, I see. Perhaps we could reword this to `Manage the runlevel of services on CentOS 6`?

Reply:
@sbrl sounds good to me. Updated